package-lock.json

```diff
@@ -20049,6 +20072,16 @@
         "uuid": "^3.3.2"
       },
       "dependencies": {
+        "form-data": {
```

Comment:
Likewise, this is a nested dependency of `jsdom` which uses `request` somewhere under the hood.